src/commands/ImportCommand.js

```diff
@@ -105,24 +117,39 @@ export default class ImportCommand extends Command {
     return ChildProcessUtilities.execSync(cmd, args, this.externalExecOpts);
   }
 
+  createPatchForCommit(sha) {
+    let patch = null;
+    if (this.options.flatten) {
+      const diff = this.externalExecSync("git", ["log", "--reverse", "--first-parent", "-p",
```

Comment:
I'd appreciate if the array of args was spread one-per-line for readability, thanks.